experimental/cuda2/pending_queue_actions.c

```diff
@@ -212,8 +338,32 @@ static const iree_hal_resource_vtable_t
         .destroy = iree_hal_cuda2_pending_queue_actions_destroy,
 };
 
-// Performs copy of the given |in_list| to |out_list| to retain the semaphore
-// and value list.
+// Copies of the given |in_list| to |out_list| to retain the command buffer
+// list.
+static iree_status_t iree_hal_cuda2_copy_command_buffer_list(
+    iree_host_size_t command_buffer_count,
+    iree_hal_command_buffer_t* const* in_list, iree_allocator_t host_allocator,
+    iree_hal_command_buffer_t* const** out_list) {
```

Comment:
I think you can drop the const here on out_list - fewer casts are always better (as a human reading would need to know that the const list isn't really a const list in some cases, and there shouldn't be any cost to it not being const when it really isn't)

Reply:
Yeah makes sense. Dropped.